### DISCUSSION TITLE
classes/class_popup

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#241] [User-contributed notes]
# classes/class_popup

Inherits: Window< Viewport< Node< Object Inherited By: PopupMenu, PopupPanel Base class for contextual windows and panels with fixed position. Description: Popup is a base class for contextual wind...

https://docs.godotengine.org/en/stable/classes/class_popup.html

**Comment by trevordane** [comment]
Popups start invisible and will need to be made visible to be seen. This can be done with popup() which is a method found under the Window class.

Much thanks to Benight from discord for figuring this out. :)

**Reply by Much0L0c0** [reply]
This information should be included in the description of both Popup and PopupMenu.
Took me a bit to even get started on understanding both.

**Reply by Calinou** [reply]
Pull request opened:

- https://github.com/godotengine/godot/pull/111963
